Jenkinsfile

```diff
@@ -14,36 +14,27 @@ node {
 
 def runStages() {
 
-    def label = "test-${UUID.randomUUID().toString()}"
-
-    podTemplate(
-        cloud: "cmqe",
-        name: "compliance-backend-test",
-        label: label,
-        inheritFrom: "compliance-backend-test",
-        serviceAccount: pipelineVars.jenkinsSvcAccount
+    openShift.withNode(
+        cloud: "upshift",
+        namespace: "insights-qe-ci",
+        yaml: "openshift/Jenkins/slave_pod_template.yaml",
+        image: "jenkins-slave-base-centos7-ruby25-openscap:latest",
```

Comment:
can we use rhel or UBI?

Reply:
This image inherits `docker-registry.engineering.redhat.com/centralci/jnlp-slave-base:1.5` which is required to use inside VPN.